instance/models/server.py

```diff
@@ -260,14 +260,20 @@ class OpenStackServer(Server):
     """
     A Server VM hosted on an OpenStack cloud
     """
+    openstack_region = models.CharField(max_length=16, blank=False)
```

Comment:
@pomegranited We generally follow the pattern that we use the settings on the instance to configure the next appserver that gets spawned.  All settings get copied to the appserver and become immutable (at least in theory), so we have a history of the settings we used in the past.  So the obvious way to select the region for the next appserver would be to set the region on the instance and then spawn an appserver.  However, we departed from the usual pattern by not copying the OpenStack parameters to the appserver; hence the comment I linked in the author concerns.